src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -165,9 +165,9 @@
   "text.autoconfig.skyblocker.option.locations.spidersDen.relics.highlightFoundRelics": "Highlight found relics",
   "text.autoconfig.skyblocker.option.locations.dungeons": "Dungeons",
   "text.autoconfig.skyblocker.option.locations.dungeons.secretWaypoints": "Dungeon Secret Waypoints",
+  "text.autoconfig.skyblocker.option.locations.dungeons.secretWaypoints.enableRoomMatching": "Enable Room Matching",
+  "text.autoconfig.skyblocker.option.locations.dungeons.secretWaypoints.enableRoomMatching.@Tooltip": "Disabling this option can save around 20 MB of ram, but Secret Waypoint and some puzzle solvers require this option to be enabled.",
```

Comment:
I would capitalize ram and add bolding to the part that mentions puzzle solvers requiring this